diff --git a/Makefile b/Makefile
index 463916a2..1037d7fd 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 export BASE_SPACE=$(shell pwd)
 export BUILD_SPACE=$(BASE_SPACE)/build
 
-VERSION = $(shell echo `git describe --tag --dirty``git status --porcelain 2>/dev/null| grep -q "^??" &&echo '-untracked'`)
+VERSION = $(shell echo `git describe --tag ``git status --porcelain 2>/dev/null| grep -q "^??" &&echo '-untracked'`)
 VERSION := $(shell echo ${VERSION} | sed -e "s/^v//")
 nightly-release: VERSION := $(shell echo ${VERSION}-nightly-build)
 # In case building outside of a git repo, use the version presented in the CWAGENT_VERSION file as a fallback
