diff --git a/Makefile b/Makefile
index 88d42dfe4..06817adb3 100644
--- a/Makefile
+++ b/Makefile
@@ -103,6 +103,12 @@ else ifeq ($(OS_ID)-$(OS_VERSION_ID),debian-11)
 	DEB_DEPENDS += virtualenv
 	DEB_DEPENDS += clang clang-format-11
 	LIBFFI=libffi7
+else ifeq ($(OS_ID)-$(OS_VERSION_ID),debian-12)
+	DEB_DEPENDS += virtualenv
+	DEB_DEPENDS += clang-14 clang-format-14
+	# for extras/scripts/checkstyle.sh
+	export CLANG_FORMAT_VER=14
+	LIBFFI=libffi8
 else
 	DEB_DEPENDS += clang-11 clang-format-11
 	LIBFFI=libffi7
